inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -472,6 +472,16 @@ std::string get_output_precision_name(ngraph::Output<Node>& o) {
     }
 }
 
+std::string escape_delim(const std::string& name) {
+    std::string result_name = name;
+    size_t index = result_name.find(',', 0);
+    while (index != std::string::npos) {
+        result_name.replace(index, 1, "\\,");
+        index = result_name.find(',', index + 2);
+    }
+    return result_name;
+}
```

Comment:
Basically this function replaces all instances of a particular substring with another substring. Probably some existing function in IE can do this?
Anyway, I would recommend to re-factor this function to be more generic and receive a string to be updated, a sub-string to be replaced and a new sub-string to be inserted.

Reply:
No in IE we don't have such functionality.
I extended this method to support different delimiters.